cspell.json

```diff
@@ -111,7 +111,9 @@
     "undefer",
     "vectorize",
     "vectorizes",
-    "walkdir",
+    "tlsv",
+    "proto",  
```

Comment:
```suggestion
    "tlsv",
    "proto",  
```
I think you can remove these lines since they're only used inside ``` quotes which are being ignored by cspell

Reply:
Removed and Cspell is blaming me for using unknown words :(

Reply:
Added again and now its working, hes not hating me anymore

Reply:
It's only ignoring blocks that are exactly 2 lines long for some reason. I'll think of the appropriate regex some other time.
Let's just add those two words